### ISSUE TITLE
how to force form mode when navigating with arrows in a form 

### ISSUE THREAD

**Issue by sigala2z** [nvaccess/nvda#12442]
-->

### Steps to reproduce:
example: https://www.pionet.com/
screenshot :   https://i.imgur.com/HPqAmOY.png
open NVDA and navigate using only arrows to one of the  form field
when entering a field (with arrow, not with TAB) try to type letters or numbers
 
### Actual behavior:
not entering form mode , can not type 

### Expected behavior:
find a way in the code to force form mode when navigating with arrows in a form ?

#### NVDA version:
2020.1

#### Windows version:
10

**Comment by feerrenrut** [comment]
Thanks for opening an issue @sigala2z. However, this issue is lacking detail. Please edit the description to explain exactly what you do, what NVDA does (speech examples help a lot), and what you want NVDA to do (again, speech examples help a lot). Explaining why you think this behavior is better will also really help.

If this issue is not made more specific, we'll have to close it, at the moment it is too open to interpretation.
